bears/general/SpaceConsistencyBear.py

```diff
@@ -41,11 +42,21 @@ def run(self,
                 if replacement[-1] != "\n":
                     replacement += "\n"
                     result_texts.append("No newline at EOF.")
+                    additional_info_texts.append(
+                        "A trailing newline character ('\\n') is missing from "
+                        "your file. "
+                        "<http://stackoverflow.com/a/5813359/3212182> will "
+                        "give you more information about why this may or may "
+                        "not be important to you.")
 
             if not allow_trailing_whitespace:
                 replacement = replacement.rstrip(" \t\n") + "\n"
                 if replacement != line.rstrip("\n") + "\n":
                     result_texts.append("Trailing whitespaces.")
+                    additional_info_texts.append(
+                        "Your source code contains trailing whitespaces. Those "
+                        "usually have no meaning and thus are redundant and "
```

Comment:
This message seems to be repetitive because you say no meaning *and* redundant.

`Your source code contains trailing whitespaces which usually have no meaning and should be removed.` ?